addons/binding/org.openhab.binding.kvv/cfg/kvv.items

```diff
@@ -0,0 +1,39 @@
+String KVV_Volkswohnung_train0_name         "[%s]"  <kvvstation-train-name>         {channel="kvv:kvvstation:35068d67:train0_name"}
```

Comment:
what is this file for? usually we don't need .items files inside a binding. if this is only for documentation purposes, add parts of that to README.md